libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/TARGET_WIZwiki_W7500/device.h

```diff
@@ -0,0 +1,74 @@
+/* mbed Microcontroller Library
+ * CMSIS-style functionality to support dynamic vectors
+ *******************************************************************************
```

Comment:
Please review all files in hal/cmsis if they contain proper license, as noted in my previous comment for sleep.c. Same here for device.h